Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp

```diff
@@ -291,6 +291,9 @@ static void bindPulse(Vector<CString>& args)
     bindIfExists(args, pulseHomeConfigDir.get());
     bindIfExists(args, asoundHomeConfigDir.get());
 
+    // system pulseaudio service
```

Comment:
Usually comments are sentences, ending with a full stop (though not always, as shown a couple lines below).